govc/dvs/create.go

```diff
@@ -34,6 +34,8 @@ type create struct {
 	configSpec *types.VMwareDVSConfigSpec
 
 	dProtocol string
+
+	numUplinkPorts int32
```

Comment:
Can use `int` here instead of `int32`, to avoid the later conversion?

Reply:
Sounds Good, but how about we use uint since the number should never be negative?

Reply:
Sure, you can do that and use `flag.Uint()`